src/lib/containers/IconList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import IconSvg, { IconSvgOptions } from './IconSvg'
+
+export type IconListProps = {
+  iconConfigs: {
+    [index:string]: IconSvgOptions
+  }
+  iconNames: string
```

Comment:
I think this thing should have a consistent name.  It's called `dataType` in the GenericCardSchema, `dataTypeIconNames` for the value variable, and `iconNames` here.
Also, can you describe the expected interface/type (on JSON.parse).

Reply:
I standardized the names to use `dataTypeIconNames` in GenericCardSchema and value variable inside the Generic Card component. But I think `iconNames` should stay the same for the IconList component because I think the icons in the list are not necessary just for data types. So I think a more generic name like "iconNames" would be more appropriate. What do you think?

`JSON.parse` has been moved to the calling GenericCard component to avoid parsing error in the demo app. So the expected type for `iconNames` is `string[]`